server/src/main/java/org/opensearch/cluster/metadata/IndexMetadata.java

```diff
@@ -405,6 +408,39 @@ public Iterator<Setting<?>> settings() {
         Property.Final
     );
 
+    public static final Setting<String> INDEX_REMOTE_TRANSLOG_REPOSITORY_SETTING = Setting.simpleString(
+        SETTING_REMOTE_TRANSLOG_STORE_REPOSITORY,
+        new Setting.Validator<>() {
+
+            @Override
+            public void validate(final String value) {}
```

Comment:
when does this get called?

Reply:
This is called when user updates this setting.